### ISSUE TITLE
Equal consecutive learning interval collapsed during studing session.

### ISSUE THREAD

**Issue by gavenkoa** [ankidroid/Anki-Android#4488]
With Anki 2.6.1 (just updated to 2.7 and have no opportunity to check on new version) if 

* hold on *desk*
* select *Deck options*
* select *New Card*
* select *Steps (in minutes)*
* enter *5 1 1 1 1 1 1 1 20* or similar pattern
* leave out of all option menus
* start learning new card

After showing card I type *Good (1 min)*, after 1 min I have only choice to type *Good (20 min)*. All equal learning intervals collapsed into single and I am prompted to check next non-equal interval.

**NOTE** Red indicator of card in learning correctly updated after checking *Good (20 min)*.  So if I have seven 1 min intervals it decreased by 7 ))

New research http://learninglab.psych.purdue.edu/downloads/2011_Karpicke_Bauernschmidt_JEPLMC.pdf showed that there is no any improvements with using increasing delays between showing cards (for learning during the day). Only repetition count and total session duration have role (the more is better). I started discussion on this topic at  http://languagelearning.stackexchange.com/questions/2344/repetition-interval-during-initial-learning-within-a-day and https://www.reddit.com/r/Anki/comments/57zdtu/what_learning_steps_interval_you_used_for/

I try to use divide duration of my typical learning session into equal intervals and trapped to this error.

I actually use **5 15 15 15 120** pattern. I should see card 3 times during in my typical 1 hour session and later see it in the evening or next day. But I see it only once.

As word-around I set interval pattern to **5 20 15 10 120** which work as expected.

**Comment by eginhard** [comment]
I can't reproduce this, it goes through every specified step just fine (at least on the 2.8 alpha, but I don't think it was any different before).
